react/SearchContext.js

```diff
@@ -17,6 +17,7 @@ const SearchContext = ({
   queryField,
   mapField,
   orderByField,
+  hideUnavailableItems = false,
```

Comment:
```suggestion
  hideUnavailableItems,
```

Is this default value needed here? There's already the `default` in the schema, and even it it didn't fulfil, it would be falsy already